Source/Resources/Content/MSGraph/ConditionalAccess/Policies/Block Platforms Other than iOS or Android from Unmanaged Devices and Untrusted Locations.json

```diff
@@ -57,7 +57,7 @@
       ]
     }
   },
-  "description": "Blocks platforms other than iOS and Android from authenticating from off-network.",
+  "description": "What is it: Blocks platforms other than iOS and Android from authenticating from devices that are off network and not Intune managed. \n Simeon Notes: This policy helps protect your data. Application protection policies on iOS and Android have been tested and verified to protect data. Intune managed devices are still able to access data off network. Simeon's baseline assigns the policy to all users except those in the Azure AD group 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. The policy is also assigned to all applications except those listed in the config property 'AppsAllowedFromUntrustedLocations'. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy",
```

Comment:
**What does this do?**
Blocks platforms other than iOS and Android from authenticating devices that are off-network and not Intune-managed. The baseline assigns the policy to all users except those in the Azure AD group 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. The policy is also assigned to all applications except those listed in the config property 'AppsAllowedFromUntrustedLocations'. 

**Why should you use this?**
This policy helps protect your data by blocking authentication from unauthorized devices. Application protection policies on iOS and Android have been tested and verified to protect data. Intune managed devices are still able to access data off network.

**What is the end-user impact?**
Users are restricted to using iOS and Android platforms for authenticating devices. @metamenos platforms or applications?

**Learn more**
[How you can protect app data](https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy)